### ISSUE TITLE
You may have encountered a bug in the Ruby interpreter or extension libraries

### ISSUE THREAD

**Issue by NetanelBasal** [mperham/sidekiq#2365]
When i am run ruby version 2.2.1p85 (2015-02-26 revision 49769) [x86_64-darwin14] and then run the command bundle exec sidekiq i am getting this error:

```ruby
You may have encountered a bug in the Ruby interpreter or extension libraries.
Bug reports are welcome.
```

**Comment by mperham** [comment]
That's not enough information to figure out what's wrong.  We need a backtrace or crash dump.

**Comment by NetanelBasal** [comment]
When i switch to version 2.1.5 the problem solved, but there is still a problem with 2.2.1, you can check it yourself.
